lib/ruby_indexer/lib/ruby_indexer/entry.rb

```diff
@@ -84,6 +84,8 @@ class Parameter
       sig { returns(Symbol) }
       attr_reader :name
 
+      alias_method :decorated_name, :name
```

Comment:
We should add a comment explaining the difference with `name` and when one should be used or the other.

Reply:
Added comments to both.

Reply:
Should the keyword parameter `foo` be decorate as `foo:`?

Reply:
Ah, yeah. Good catch.